packages/store/src/storage.ts

```diff
@@ -1044,6 +1046,8 @@ class Storage {
     // at this point document is either created or existed before
     const [firstAttachmentKey] = Object.keys(_attachments);
     const buffer = this.getBufferManifest(body._attachments[firstAttachmentKey].data as string);
+    const readable = Readable.from(buffer);
```

Comment:
I'd move this inside of `getTarballStats` since is the unique place is being used, otherwise would create an unused variable in most of the cases.